scripts/apps/users/views/user-preferences.html

```diff
@@ -423,15 +423,22 @@ <h4 class="sd-heading sd-text-align--left sd-text--sans sd-heading--h4 sd-margin
                     </div>
                 </li>
 
-                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start">
-                    <h3 id="macros-config" class="sd-heading sd-text-align--left sd-text--sans sd-heading--h3" translate>{{preferences['macro_config'].label}}</h3>
-                    <div class="sd-container sd-container--flex sd-container--gap-none sd-container--direction-column sd-radius--medium sd-panel-bg--000 sd-shadow--z2 sd-padding--3 sd-state--focus sd-margin-b--1">
-                        <div sd-info-item>
-                            <div ng-repeat="(key, field) in preferences.macro_config.fields">
-                            <label class="form-label" translate>{{ key }}</label>
-                            <input type="text" ng-model="preferences['macro_config'].fields[key]" />
+                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start"></li>
```

Comment:
what's the purpose to have an empty `<li>` tag?

Reply:
@fritzSF suggested these changes, and I’ve applied them as is.